libs/cardano-ledger-test/src/Test/Cardano/Ledger/Constrained/Conway/LedgerTypes/Specs.hs

```diff
@@ -522,27 +583,13 @@ newEpochStateSpecUnit pp =
 
 -- ===============================
 
--- ===================
-
 dRepToCred :: DRep -> Maybe (Credential 'DRepRole)
 dRepToCred (DRepKeyHash kh) = Just $ KeyHashObj kh
 dRepToCred (DRepScriptHash sh) = Just $ ScriptHashObj sh
 dRepToCred _ = Nothing
 
--- ===================================
+credToDrep :: Credential 'DRepRole -> DRep
+credToDrep (KeyHashObj kh) = DRepKeyHash kh
+credToDrep (ScriptHashObj sh) = DRepScriptHash sh
```

Comment:
This is unused and is not necessary. We have a `DRepCredential` for this already: https://github.com/IntersectMBO/cardano-ledger/blob/65ef348db5b1000df82264dfe74a2443425556d7/libs/cardano-ledger-core/src/Cardano/Ledger/DRep.hs#L116-L117
```suggestion
```